tensorflow/lite/kernels/activations.cc

```diff
@@ -333,12 +333,9 @@ TfLiteStatus ReluEval(TfLiteContext* context, TfLiteNode* node) {
   TfLiteTensor* output = GetOutput(context, node, 0);
   switch (input->type) {
     case kTfLiteFloat32: {
-      size_t elements = input->bytes / sizeof(float);
-      float* in = input->data.f;
-      float* in_end = in + elements;
-      float* out = output->data.f;
-      for (; in < in_end; in++, out++) *out = std::max(0.f, *in);
-      return kTfLiteOk;
+      // refering to the optimized implementation
+      optimized_ops::Relu(GetTensorShape(input), GetTensorData<float>(input),
+                          GetTensorShape(output), GetTensorData<float>(output));
```

Comment:
return kTfLiteOk?

Reply:
@renjie-liu , thanks for pointing this out, i have updated the code as per the comments, Kindly check and approve.

Regards
Amit